tests/integration/test_prometheus_scrape_multiunit.py

```diff
@@ -48,13 +48,14 @@
 idle_period = 90
 
 
+@pytest.mark.skip(reason="xfail")
 async def test_setup_env(ops_test: OpsTest):
     await ops_test.model.set_config(
         {"logging-config": "<root>=WARNING; unit=DEBUG", "update-status-hook-interval": "60m"}
     )
 
 
-@pytest.mark.xfail
+@pytest.mark.skip(reason="xfail")
```

Comment:
What's the difference between `@pytest.mark.xfail` and `@pytest.mark.skip(reason="xfail")`?
And why do we suddenly have so many skips?

Reply:
`@pytest.mark.xfail` executes the tests, but ignores the end result.
`@pytest.mark.skip` skips the tests, just like commenting it out.

The purpose of `xfail` is, generally, to mark tests that **should**, but don't pass. Ideally, you would temporarily mark something with `xfail`, understand what the problem is, and fix it.

If we had a test suddenly failing only in CI, we'd probably `xfail` it until we fix it.

However, what we do with `xfail` is use it to essentially "not care about the result of a test" without the guilt of removing it. Our oldest `xfail` of this set is from November 2022, and to my knowledge we never went to revisit an `xfail`ing test, to the point I'm sure no one would be able to tell you if they're passing or not.

Instead of deleting them though, we can just `skip` until we decide what to do with them; it's hard to see (due to `charmcraft pack` taking 10 minutes extra), but we improved the itests execution time by about 10 minutes.